operator/init_test.go

```diff
@@ -3,6 +3,6 @@ package operator
 import "github.com/sirupsen/logrus"
 
 func init() {
-	logrus.SetLevel(logrus.InfoLevel)
+	logrus.SetLevel(logrus.TraceLevel)
```

Comment:
I wouldn't change that in master.

Reply:
I just saw that the `tee/prototype/init_test.go` is set to `TraceLevel`, we could also change it there.